### ISSUE TITLE
Sample identifiers

### ISSUE THREAD

**Issue by cpavloud** [emo-bon/observatory-hcmr-1-crate#13]
It is not practical to have identifiers such as EMOBON_HCMR-1_Wa_210917_0.2um_blank1 and EMOBON_HCMR-1_Wa_210917_0.2um_blank2

**Comment by cpavloud** [comment]
I would propose to use the names EMOBON_HCMR-1_Wa_210917_0.2um_blank**_**1 and EMOBON_HCMR-1_Wa_210917_0.2um_blank**_**2 instead. 
@kmexter what do you think?

**Comment by kmexter** [comment]
urm, why is it not practical? ...PS I think your suggestion got malformed, can you write it again without formatting imposed?
the thing is that there are 2 blanks for the same sample, so that was my way of making them separate

**Comment by cpavloud** [comment]
Yes, we need the 1 and 2, but it would be easier I think to have them _1 and _2 and not directly linked to **blank** (for subsequent data processing). 
Something to think about.

**Comment by kmexter** [comment]
oh right, OK, that should be doable. I will have a think on the best way to do that in the googlesheet equation for that column

**Comment by kmexter** [comment]
Need a bit of a think here. It is good to have just "blank" as the replicate type, but I am not sure how a google equation can add a _1 and _2 if there are more than one blank, to the material sample ID
I throw this back at you, @cpavloud, to ask: should all blanks have the _1 and _2 etc, or only those that are multiple? Mostly it is just one blank, but several stations have submitted more. If we are going to have a _2 then we should have a _1 everywhere, but the easiest way to do that, then, is to have these 1s and 2s in another column. Or, we have a column for "replicate type" being "sample" or "blank" and then replicate number, which does not change except the blanks can now get 1s and 2s. WDYT?

**Comment by cpavloud** [comment]
I see that the way the source_mat_id for the blanks is filled is taking information from column (M) which is the replicate column. 
So if instead of **blank1** and **blank2** we write **blank_1** and **blank_2**, we should be ok.

**Comment by cpavloud** [comment]
I would say, blank_1 for all, even if there is no blank_2

**Comment by kmexter** [comment]
Ok, but then that will require changing the existing entries which just say "blank". OR, we can keep them like this in the logsheets (blank, or blank1+blank2) and change it on the transforming (if en event has one blank change blank to blank1)?

**Comment by kmexter** [comment]
OK, I think I can take on the effort of changing this in the logsheets, but then we need to tell people of this (among several other) updates we have made to the logsheets, or we will have many new mistakes creeping in with new events added to the logsheets

**Comment by kmexter** [comment]
@cymon I have changed the sediment logsheets, see e.g. https://docs.google.com/spreadsheets/d/1zc0bZdpl-Eoi35lI_5BGkElbscplyQRyNPLkSgeEyEQ/edit?gid=874595611#gid=874595611. Check if that affects any of your workflows

@laurianvm can you also check if that changes any of your workflows?

@bulricht likewise

**Comment by cymon** [comment]
@kmexter Well, it doesn't because of this issue: [Change the format of the sampling:replicate field from Number:auto to Number:Plain Text](https://github.com/emo-bon/observatory-profile/issues/15)

Did you make other changes to fields apart from "replicate"?

[for example this is the same sheet you link to above - it's missing the "blank_*"'s](https://docs.google.com/spreadsheets/d/1HuXHiUJICZrmCrJ4EZDyU5aSCMzDAc1cy_tne5YVPTg/gviz/tq?tqx=out:csv&sheet=sampling)

**Comment by kmexter** [comment]
Done the water also now. I really think next time we shoud make the mat_samp_id just a plain iterator - HCMR0001, 0002, 0003, etc, to avoid all these problems!

I say again _please update the handbook_ and _please send new instructions to the stations_ for the new recommendations - blank_1, not using > in the filter columns, and whatever else we have done. Otherwise, all the same mistakes will come in to the logsheets for entries for now on.

**Comment by kmexter** [comment]
The only thing I did not do was to update the definition because I encountered problem with the Updated definitions tab in the sediment logsheetes (whihch I emaiedl @melanthia and @melinalou about) so can I ask HCMR EMOBON HQ to do that? change the definition in all water and sediment tabs to 

"Indicates the replicate code of the material sample. For replicate 1, add "1", for replicate 2, add "2", etc. For blank samples, add "blank_1" for the first (even if there is never a second), and "blank_2" etc for subsequent blanks from the same event."

**Comment by kmexter** [comment]
@cymon ... the blank_1 and blank_2 are definitely in the googlesheet itself
we also changed - or will change - the material sample replicate for sediment by adding in the filter size. I think that is all the changes we made to that.

**Comment by cpavloud** [comment]
So, all of the sample_ids are corrected?

**Comment by cymon** [comment]
@cymon ... the blank_1 and blank_2 are definitely in the googlesheet itself
@kmexter yes, I know, we've already had this conversation :) but I've explained why they are missing from the CSV and how to fix it

Also if you have changed the format of the "source_mat_id" field in the sampling sheets, they also need to be changed to match in the ["Batch_run_information"](https://github.com/emo-bon/sequencing-data/blob/main/shipment/batch-002/run-information-batch-002.csv) sheets and elsewhere.

**Comment by kmexter** [comment]
for the blanks yes, for the sediment I don't know - that is for HCMR to do and I don't think that has been done yet

**Comment by kmexter** [comment]
yes, I will do the batch information - new issue
HCMR have to do the other one - Ive already spent more time than I can on this. you should raise an issue on them, and assign it to them, if you have not already, as it is easy to loose track

**Comment by melanthia** [comment]
Hi all,I will do the changes tomorrow morning, I'm off the office now. I hope I can actually do it actually and not have locked cells again. Cheers, Melanthia Στάλθηκε από Outlook για AndroidΑπό: Katrina Exter ***@***.***>Eστάλη: Πέμπτη, Σεπτεμβρίου 5, 2024 4:28:52 μ.μ.Προς: emo-bon/observatory-hcmr-1-crate ***@***.***>Κοιν.: Melanthia Stavroulaki ***@***.***>; Assign ***@***.***>Θέμα: Re: [emo-bon/observatory-hcmr-1-crate] Sample identifiers (Issue #13)
The only thing I did not do was to update the definition because I encountered problem with the Updated definitions tab in the sediment logsheetes (whihch I emaiedl @melanthia and @melinalou about) so can I ask HCMR EMOBON HQ to do that? change the definition in all water and sediment tabs to
"Indicates the replicate code of the material sample. For replicate 1, add "1", for replicate 2, add "2", etc. For blank samples, add "blank_1" for the first (even if there is never a second), and "blank_2" etc for subsequent blanks from the same event."

**Comment by kmexter** [comment]
@melanthia and @melinalou did you manage to change the definitions? IF yes, you can close this issue
